### PR TITLE
Expand the README with example data.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +43,62 @@ then iterates over that vector calling a method:
...
+# Example results
+
+`vtable_bench` takes (optionally) 3 arguments:
+
+```
+$ cargo run --release --bin vtable_bench -- -h
+Usage: vtable_bench [-h] [<#reps> <#iters> <#vec size>]
+```
+
+From right to left (since it's easier to understand that way): `vec size` is how
```
Comment (vext01) [inline README.md]: A bit of a nitpick perhaps, but I found this a little jarring:

 > `vec size` is how many elements the vector that's iterated will contain

How about:

>  `vec size` determines the size of the vectors.

(You've already said you will iterate them earlier)

File: README.md [hunk 2]
```diff
@@ -43,3 +43,62 @@ then iterates over that vector calling a method:
...
+many elements the vector that's iterated will contain; `iters` is how many times
+the `for` loops that reads from the vector is read from; and `reps` is how many
```
Comment (vext01) [inline README.md]: Similarly with:

> `iters` is how many times the `for` loops that reads from the vector is read from

How about:

> `iters` determines the number of times each vector is iterated.

File: README.md [hunk 3]
```diff
@@ -43,3 +43,62 @@ then iterates over that vector calling a method:
...
+times the benchmark is executed from scratch (creating a fresh Unix process each
+time).
+
+Here are examples runs with a gradually increasing `vec size` (and a decreasing
```
Comment (vext01) [inline README.md]: examples -> example

**Review by vext01 — COMMENTED** [review]
A few nitpicks.

File: README.md (hunk 1 above)
Comment (ltratt) [reply README.md]: "Determine" is a vague word IMHO: isn't it clearer to say explicitly that it's how many elements the vector will contain?

File: README.md (hunk 3 above)
Comment (ltratt) [reply README.md]: Oops!

File: README.md (hunk 1 above)
Comment (vext01) [reply README.md]: That's fine, but I found the original wording awkward.

So you could say, for example:

> The iterated vectors will contain `vec_size` elements.

Comment (ltratt) [reply README.md]: I'm not sure I've made it better, but see if you prefer the new phrasing.

**Comment by vext01** [comment]
I think it's the best we will get. You wanna squash?

**Comment by ltratt** [comment]
Squashed.

**Review by vext01 — APPROVED** [review]

**Pull request merged**